contributing/documentation-styleguide.md

```diff
@@ -109,36 +113,50 @@ When writing a style guide, consider:
 Use hyphens to separate words in the file name and save it as a `.md` file in the relevant directory. If you need to add supporting images, save them in an `/images` directory within the same directory where you wish to store the style guide.
 
 #### Other Internal Documentation
```

Comment:
Are these headings at the correct hierarchy now you've updated the other ones?

![image](https://user-images.githubusercontent.com/44157880/196698244-cc758ead-a763-4068-9845-63279f053e8b.png)

Reply:
Just gone back through the files and realised some of the levels were incorrect in `documentation-styleguide.md`. I will correct these 👍🏼